### PR TITLE
Convert RefundPayment state values to lowercase

### PR DESCRIPTION
**Pull request by diimpp**
Fixes https://github.com/Sylius/RefundPlugin/issues/188
Supersedes https://github.com/Sylius/RefundPlugin/pull/190

### CODE REVIEW

File: src/Resources/views/Common/Label/refundPaymentState.html.twig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{%
+    set viewOptions = {
+        completed: { icon: 'adjust', color: 'green' },
+        new: { icon: 'clock', color: 'olive' },
+    }
+%}
+
+{% set value = 'sylius.ui.' ~ data %}
```
Comment (diimpp) [inline src/Resources/views/Common/Label/refundPaymentState.html.twig]: Is it alright to re-use `sylius.ui` translations for `new` and `completed` states? Should I replace it with english only `sylius_refund.ui` values?

File: src/Resources/views/Common/Label/refundPaymentState.html.twig [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{%
+    set viewOptions = {
+        completed: { icon: 'adjust', color: 'green' },
```
Comment (diimpp) [inline src/Resources/views/Common/Label/refundPaymentState.html.twig]: Colors and icons are re-used from regular payment widget. Looks to me better balanced.

File: src/Resources/views/Order/Admin/RefundPayment/list.html.twig [hunk 3]
```diff
@@ -17,7 +17,7 @@
                 <tr>
                     <td>{{ money.format(refund_payment.amount, refund_payment.currencyCode) }}</td>
                     <td>
-                        {% include [('@SyliusRefundPlugin/RefundPayment/Label/State/' ~ refund_payment.state ~ '.html.twig'), '@SyliusUi/Label/_default.html.twig'] with {'value': refund_payment.state} %}
+                        {% include '@SyliusRefundPlugin/Common/Label/refundPaymentState.html.twig' with {'data': refund_payment.state} %}
```
Comment (diimpp) [inline src/Resources/views/Order/Admin/RefundPayment/list.html.twig]: `include` is redesigned in fashion of SyliusAdmin/SyliusShop paymentState label.

**Comment by diimpp** [comment]
@Zales0123 

```twig
 {% if refund_payment.state != 'sylius_refund.ui.completed'|trans %}
```

image

I will just leave it here as thinking material :D

**Comment by diimpp** [comment]
@lchrusciel @Zales0123 hey guys, any questions? Can it be merged?

**Comment by lchrusciel** [comment]
It can be merged, but we need to add it to UPGRADE.md. Can you do it?

**Comment by diimpp** [comment]
@lchrusciel done

**Comment by lchrusciel** [comment]
Thank you, Dmitri! :tada:

**Comment by diimpp** [comment]
Great, thank you! :)

**Pull request merged**